3_visualize.R

```diff
@@ -20,13 +20,48 @@ p3_targets <- list(
   tar_target(p3_drought_learner_viz01_png,
              drought_lrnr_viz01(p2_droughts_learner_viz_df,
                                 p2_streamflow_learner_viz_df,
-                                out_png = "3_visualize/out/drought_learner_01.png"),
+                                out_png = "src/assets/images/threshold-chart/a.png"),
```

Comment:
I know this was my naming convention, but in the long term I agree with using a numeric system for the frame ordering. As we evolve this, we should build out the array of text so that it has a unique id and order value for each frame.